README.md

```diff
@@ -3,10 +3,12 @@
 # Math-Magicians
 "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) build with React a JavaScript Library. It allows users to: Make simple calculations. Read a random math-related quote.
 
-![screenshot](images/calculator.png)
+![screenshot](images/fullPage.png)
 
 ## Features:
 - Displays the calculator's interface
+- Enables you to carryout basic arithmetic operations
+- Read a random math-related quote.
```

Comment:
- [ ] **OPTIONAL**
Kindly update the Live Demo Link attached to your README file to the latest version of the app for a completely professional README file and repo 👍 
_Please note that such small things might leave big impressions on future viewers._ 😄